src/web/mjs/connectors/ComicTrail.mjs

```diff
@@ -39,13 +39,15 @@ export default class ComicTrail extends SpeedBinb {
         let request = new Request(this.url + manga.id, this.requestOptions);
         let data = await this.fetchDOM(request, this.queryChapters);
         return data.map(element => {
-            let part = "";
-            if (!/\d{4}-\d{1,2}-\d{1,2}/.test(element.children[1].firstElementChild.firstElementChild.children[1].textContent)){
-                part = ` ${element.children[1].firstElementChild.firstElementChild.children[1].textContent.trim()}`;
+            let id = "";
+            if (element.querySelector('a.button.is-read') != null){
```

Comment:
This should never be `null`, so the `if` check is obsolete, otherwise there would be chapters with empty IDs which will throw an exception sooner or later.

Reply:
its there because a lot of chapters don't have IDs right now (u gotta pay for them)
i don't know how to deal with them